modules/importing-repositories/procedure-managing-the-imported-repository-list.adoc

```diff
@@ -0,0 +1,30 @@
+[id="managing-the-added-repositories]
+= Managing the added repositories
+
+.Prerequisites
+* You have xref:importing-multiple-github-repositories[imported GitHub repositories].
+
+
+.Procedure
+. Click *Bulk Import* in the left sidebar to display all the current repositories that are being tracked as Import jobs, along with their status.
+
+Added:: The repository is added to the {product-short} catalog after the import pull request is merged or if the repository already contained a `catalog-info.yaml` file during the bulk import. 
+This can be the case after the import PR has been merged, or if the repository already had a `catalog-info.yaml` when it was bulk-imported.
```

Comment:
This sentence is being repeated. Please remove this sentence

Reply:
```suggestion
```